queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -137,6 +168,37 @@ def _get_pdfs(
             )
             attach_order += 1
 
+        # add receipt
```

Comment:
receipt will be attached only in PAID status

Reply:
okay, I will remove this from the email template, like the one above

Reply:
confirm with Mihai/Jacqueline before removing

Reply:
The user should receive a receipt by email once they submit the application on the bc registries system even if it's future effective because we take the payment immediately even if the continuation is future effective.

Reply:
@jacqueline-williams-549 yes, we sent out a receipt in the first email, please refer to the comment above (just posted)